src/colors/colors.css

```diff
@@ -1,12 +1,38 @@
 :root {
-  --omrs-primary-blue: #1e4bd1;
-  --omrs-text-black: #121212;
-  --omrs-inactive-grey: #3c3c43;
-  --omrs-background-canvas: #fafafa;
-  --omrs-critical-alert-red: #b50706;
-  --omrs-success-alert-green: #006800;
-  --omrs-warning-alert-brown: #6d5d01;
-  --omrs-interaction-plus-one: #193faf;
-  --omrs-interaction-minus-two: rgba(73, 133, 224, 0.12);
-  --omrs-ink-low-contrast: rgba(60, 60, 67, 0.3);
+  --omrs-color-inactive-grey: #3c3c43;
+  --omrs-color-primary-blue: #1a73e8;
+  --omrs-color-text-black: #212121;
```

Comment:
We should update https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/typography/typography.css#L24 to have the new name for the color.

That code was recently merged in in #14, and needs to be updated with this PR